File: BATCH_21/js and python/c2_loops.js
// print 0 to 9

for (let i = 0; i < 10; i = i + 1) {
    console.log(i)
}

//  print even numbers from 0 to 20
for (let i = 0; i < 21; i = i + 2) {
    console.log(i)
}

// print table of 2
for (let i = 1; i <= 10; i = i + 1) {
    console.log("2 x ", i, " = ", i * 2)
}

// Print prime number
let num = 45;
let isPrime = true;
for (let i = 2; i < num; i++) {
    if (num % i == 0) {
        isPrime = false;
    }
}

if (isPrime == false) {
    console.log("Not Prime!")
} else {
    console.log("Prime!")
}

File: BATCH_21/js and python/c2_loops.py
'''
# print 0 to 9
for i in range(10):
    print(i)

#  print even numbers from 0 to 20
for i in range(0,20,2):
    print(i)

# print table of 2
for i in range(10):
    print("2 x " , i , " = " , i * 2)
'''

# Print prime number
num = 45
isPrime = True
for i in range(2,num): 
    if (num % i == 0):
        isPrime = False

if (isPrime == False):
    print("Not Prime!")
else:
    print("Prime!")